sql/qaqc/qaqc_app_additions.sql

```diff
@@ -15,3 +15,27 @@ SELECT a.job_number,
 	END) as classa_net_mismatch
 INTO qaqc_app_additions
 FROM FINAL_devdb a;
+
+ALTER TABLE qaqc_app_additions ADD manual_hny_match_check INT;
+WITH manual_hny_match_check AS (
+    SELECT 
+    job_number
+    FROM corr_hny_matches 
+    WHERE 
+    action = 'add' 
+    AND
+    hny_id IN (SELECT hny_id FROM hny_no_match)
+)
+UPDATE qaqc_app_additions 
+SET manual_hny_match_check = (
+    CASE 
+        WHEN qaqc_app_additions.job_number NOT IN (SELECT job_number FROM manual_hny_match_check) THEN 0 
+        ELSE 1 
+    END);
+
+ALTER TABLE qaqc_app_additions ADD manual_corrections_not_applied INT;
+UPDATE qaqc_app_additions
+SET manual_corrections_not_applied=(
+    CASE 
+        WHEN qaqc_app_additions.job_number IN (SELECT job_number FROM corrections_not_applied) THEN 1 ELSE 0 
```

Comment:
My concern here is that we shouldn't just use the job numbers in the existing qaqc_app_additions, as it is completely based on classa_net_mismatch check. There are 477 records with manual_corrections_not_applied=1 in the final qaqc_app table, however there are distinct 479 records in the corrections_not_applied table. 

While the inner query for the manual_hny_match_check does return nothing on my machine, which doesn't affect the result anyway.

Reply:
good call let me think about how to make this better. Probably easy solution is to switch over the final devdb

Reply:
On second thought about this, isn't how `classa_net_mismatch` table constructed ensured that it has all job numbers from `final_devdb`? Let me know if that's a misreading on my part. @Oysters1874

Reply:
Oh yea, I just took another look at the code. classa_net_mismatch has all job_numbers already, sorry about this.

Reply:
@Oysters1874 I ran a check on what job number are in manual corrections but no longer in the `final_devdb` and it turns out that there is exactly two distinct job numbers (5 records). So it all checked out!

Reply:
Awesome. Then I think it is good to got merged in.